File: 程式語言 V2答案檔/Python/607.py
flower = ['S', 'H', 'D', 'C']
cards = input().split(' ')
if flower.index(cards[0][0]) < flower.index(cards[1][0]):
    print(cards[0] + ' > ' + cards[1])
elif flower.index(cards[0][0]) > flower.index(cards[1][0]):
    print(cards[0] + ' < ' + cards[1])
else:
    if int(cards[0][1:]) < int(cards[1][1:]):
        print(cards[0] + ' < ' + cards[1])
    elif int(cards[0][1:]) > int(cards[1][1:]):
        print(cards[0] + ' > ' + cards[1])
    else:
        print(cards[0] + ' = ' + cards[1])

File: 程式語言 V2答案檔/C#/607.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class TQC607
{
    public static void Main(string[] args)
    {
        string input = Console.ReadLine();
        List<string> cards = new List<string> ();
        cards = input.Split(new char[] { ' ' }).ToList<string>();
        char[] colors = { 'C', 'D', 'H', 'S' };
        if (Array.IndexOf(colors, cards[0][0]) > Array.IndexOf(colors, cards[1][0]))
        {
            Console.WriteLine($"{cards[0]} > {cards[1]}");
        }
        else if (Array.IndexOf(colors, cards[0][0]) < Array.IndexOf(colors, cards[1][0]))
        {
            Console.WriteLine($"{cards[0]} < {cards[1]}");
        }
        else
        {
            int num0 = Convert.ToInt32(cards[0].Substring(1, cards[0].Length - 1));
            int num1 = Convert.ToInt32(cards[1].Substring(1, cards[1].Length - 1));
            if (num0 > num1) Console.WriteLine($"{cards[0]} > {cards[1]}");
            else if (num0 < num1) Console.WriteLine($"{cards[0]} < {cards[1]}");
            else Console.WriteLine($"{cards[0]} = {cards[1]}");
        }
    }
}